analyzers/tests/SonarAnalyzer.UnitTest/TestCases/RedundantConditionalAroundAssignment.CSharp9.cs

```diff
@@ -80,12 +80,12 @@
 
 SomeClass someClass = new SomeClass() { SomeField = 42 };
 
-if (someClass.SomeField == 42) // FN
+if (someClass.SomeField != 42) // Noncompliant
```

Comment:
why remove the previous test?

Reply:
It was actually not a valid test case and not a FN. You can see documented [here](https://github.com/SonarSource/sonar-dotnet/blob/9e019155aa3cd342adcd106a952304e790a7b328/analyzers/tests/SonarAnalyzer.UnitTest/TestCases/RedundantConditionalAroundAssignment.cs#L80). The rule is about redundant conditional, but with the previous example it was actually the assignment that was redundant.